src/sqlfluff/dialects/dialect_bigquery.py

```diff
@@ -440,7 +434,36 @@ class ExtractFunctionNameSegment(BaseSegment):
     """
 
     type = "function_name"
-    match_grammar: Matchable = Ref("ExtractFunctionName")
+    match_grammar: Matchable = StringParser(
+        "EXTRACT",
+        CodeSegment,
+        name="function_name_identifier",
+        type="function_name_identifier",
+    )
+
+
+class NormalizeFunctionNameSegment(BaseSegment):
+    """NORMALIZE function name segment.
+
+    Need to be able to specify this as type function_name
```

Comment:
```suggestion
    Need to be able to specify this as type function_name_identifier
```

Reply:
Fixed in https://github.com/sqlfluff/sqlfluff/pull/3086/commits/b49c64db0530f1a9abff439b77593d792063f1cf